srcpkgs/dwarffortress/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'dwarffortress'
+pkgname=dwarffortress
+version=0.44.12
+revision=1
+archs="x86_64"
+create_wrksrc=yes
+depends="gtk+>=2 SDL>=1.2 SDL_ttf SDL_image libGL glu"
```

Comment:
don't need to specificy versoins since gtk+ and SDL are always bigger than the versions being specified